Tests/test_Nexus.py

```diff
@@ -438,7 +438,72 @@ def test_NexusComments(self):
                           (None, 0.20039426358269385, None, '[&height_95%_HPD={0.3110921040545068,0.38690865205576275},length_range={0.09675588357303178,0.4332959544380489},length_95%_HPD={0.16680375169879613,0.36500804261814374}]'),
                           ('9', 0.055354097721950546, None, '[&rate_range={1.3E-5,0.10958320752991428},height_95%_HPD={0.309132419999969,0.3091324199999691},length_range={3.513906814545109E-4,0.4381986285528381},height_median=0.309132419999969,length_95%_HPD={0.003011577063374571,0.08041621647998398}]'),
                           ('5', 0.055354097721950546, None, '[&rate_range={1.3E-5,0.10958320752991428},height_95%_HPD={0.309132419999969,0.3091324199999691},length_range={3.865051168833178E-5,0.4391594442572986},height_median=0.309132419999969,length_95%_HPD={0.003011577063374571,0.08041621647998398}]')])
+class TestSelf(unittest.TestCase):
+    def doctest(self):
+        print("Repeated names without a TAXA block")
+    handle = StringIO("""#NEXUS
+    [TITLE: NoName]
+    begin data;
+    dimensions ntax=4 nchar=50;
+    format interleave datatype=protein   gap=- symbols="FSTNKEYVQMCLAWPHDRIG";
+    matrix
+    CYS1_DICDI          -----MKVIL LFVLAVFTVF VSS------- --------RG IPPEEQ---- 
+    ALEU_HORVU          MAHARVLLLA LAVLATAAVA VASSSSFADS NPIRPVTDRA ASTLESAVLG 
+    CATH_HUMAN          ------MWAT LPLLCAGAWL LGV------- -PVCGAAELS VNSLEK----
+    CYS1_DICDI          -----MKVIL LFVLAVFTVF VSS------- --------RG IPPEEQ---X
+    ;
+    end; 
+    """)  # noqa for pep8 W291 trailing whitespace
+    for a in NexusIterator(handle):
+        print(a)
+        for r in a:
+            print("%r %s %s" % (r.seq, r.name, r.id))
+    print("Done")
 
+    print("")
+    print("Repeated names with a TAXA block")
+    handle = StringIO("""#NEXUS
+    [TITLE: NoName]
+    begin taxa
+    CYS1_DICDI
+    ALEU_HORVU
+    CATH_HUMAN
+    CYS1_DICDI;
+    end;
+    begin data;
+    dimensions ntax=4 nchar=50;
+    format interleave datatype=protein   gap=- symbols="FSTNKEYVQMCLAWPHDRIG";
+    matrix
+    CYS1_DICDI          -----MKVIL LFVLAVFTVF VSS------- --------RG IPPEEQ---- 
+    ALEU_HORVU          MAHARVLLLA LAVLATAAVA VASSSSFADS NPIRPVTDRA ASTLESAVLG 
+    CATH_HUMAN          ------MWAT LPLLCAGAWL LGV------- -PVCGAAELS VNSLEK----
+    CYS1_DICDI          -----MKVIL LFVLAVFTVF VSS------- --------RG IPPEEQ---X
+    ;
+    end; 
+    """)  # noqa for pep8 W291 trailing whitespace
+    for a in NexusIterator(handle):
+        print(a)
+        for r in a:
+            print("%r %s %s" % (r.seq, r.name, r.id))
+    print("Done")
+    print("")
+    print("Reading an empty file")
```

Comment:
This would become a third ``unittest`` method, perhaps called ``test_empty_file``?